src/App.tsx

```diff
@@ -1,39 +1,109 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import Carousel from './components/Carousel';
+import { images } from './images';
 
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
+const initialOptionsState = {
+  step: 3,
+  frameSize: 3,
+  itemWidth: 130,
+  animationDuration: 1000,
+  infinite: false,
+};
+
+export const App = () => {
+  const [options, setOptions] = useState(initialOptionsState);
+
+  const handleOptionChange = (key: string, value: number | boolean) => {
+    setOptions(prevOptions => ({ ...prevOptions, [key]: value }));
   };
 
-  render() {
-    const { images } = this.state;
+  const {
+    step, frameSize, itemWidth, animationDuration, infinite,
+  } = options;
+
+  return (
+    <div className="App">
+      <h1
+        className="App__title"
+        data-cy="title"
+      >
+        {`Carousel with ${images.length} images`}
+      </h1>
+
+      <div className="App__labels">
+        <label>
+          Step:
+          <input
+            type="number"
+            min={1}
+            max={10}
+            value={step}
+            onChange={(e) => handleOptionChange('step', +e.target.value)}
+          />
+        </label>
+
+        <label>
+          Frame size:
+          <input
+            type="number"
+            min={1}
+            max={10}
+            value={frameSize}
+            onChange={(e) => handleOptionChange(
```

Comment:
It can be improved:
```suggestion
            onChange={handleOptionChange}
```
Set names to input and use one handler

```
const handleOptionChange = (e) => {
    const { name, value } = e.target
    setOptions(prevOptions => ({ ...prevOptions, [name]: value }));
  };
  ```